Z/Zstd/build_tarballs.jl

```diff
@@ -22,7 +22,20 @@ elif [[ "${target}" == i686-*-mingw* ]]; then
     sed -ri "s/^c_args = \[(.*)\]/c_args = [\1, '-D_WIN32_WINNT=_WIN32_WINNT_VISTA']/" ${MESON_TARGET_TOOLCHAIN}
 fi
 
-meson --cross-file="${MESON_TARGET_TOOLCHAIN}" ../build/meson/
+# Meson doesn't know how to handle the linker on
+# aarch64-apple-darwin. We thus pretend to use a different linker there.
+# We are going to switch back once Meson is done.
+if [[ "${target}" == aarch64-apple-darwin* ]]; then
+    sed -i 's+aarch64-apple-darwin20-ld+aarch64-apple-darwin20-ld64.lld+' "${MESON_TARGET_TOOLCHAIN}"
+fi
```

Comment:
Would `clang_use_lld=false` help here?

Reply:
Good catch. I tried a few things but apparently missed this idea.